Description: Fix infinite recursion in SSL
 This fixes an infinite recursion in SSL when using dnspython 2.x.
Author: Tim Burke <tim.burke@gmail.com>
Origin: upstream, https://github.com/tipabu/eventlet/commit/2f9b7969f9a66a75e72908454246b88bf57fe58a
Last-Update: 2021-02-04

--- python-eventlet-0.26.1.orig/eventlet/support/greendns.py
+++ python-eventlet-0.26.1/eventlet/support/greendns.py
@@ -42,6 +42,7 @@ from eventlet.green import _socket_nodns
 from eventlet.green import os
 from eventlet.green import time
 from eventlet.green import select
+from eventlet.green import ssl
 import six
 
 
@@ -55,6 +56,7 @@ def import_patched(module_name):
         'time': time,
         'os': os,
         'socket': _socket_nodns,
+        'ssl': ssl,
     }
     return patcher.import_patched(module_name, **modules)
 
